meilenstein_02/analyse_Speichermedium/README.md

```diff
@@ -0,0 +1,108 @@
+- Werden noch weitere Funktionen als die Basics benötigt ?
+- Eventuell muss sich das Speichermedium noch um die Aufnahme der Configs für die Funktionen in den Dateien kümmern
+
+## Was sind die Ziele des Systems?
+<!-- Snow Cards können bei diesem Schritt helfen! -->
+- Es sollen die Daten von den drei Funktionen (sowie ihre eingestellte Config-Daten) gespeichert werden, jede Funktion speichert jeweils in einer eigenen Datei
+- Es ist möglich Dateien zu erstellen für verschiedene Datensätze der Funktionen von verschiedenen Versuchen
+- Es wird dem Dataclient eine Klasse mit Funktionen zur Verfügung gestellt, welche ihm ermöglichen nach Informationen, Dateien, als auch ganzen Datenordnern zu suchen
+- Das System ermöglicht es Datensätze auch wieder zu löschen
+
+## Was ist der erwartete Input?
+Es werden Datensätze von den Funktionen *Zoom Maps*, *Codecharts*, *Webcam basiertes Eyetracking* und DataClient erwartet. 
+
+## Was ist der erwartete Output?
+Es wird erwartet, dass der DataClient nach Dateien, Ordnern, als auch einzelnen Informatione fragt und diese ihm dann ausgegeben werden müssen.
+
+## Was sind Nomen Phrasen?
+<!-- Alle relevanten Sachen aufschreiben, später kann aussortiert werden! -->
+- Speicher/Schreiben
+- Lesen/Suchen
+- Löschen/Aendern
+- Sortieren
+
+## Liste der Klassen:
+<!-- Erstmal alle aufschreiben, dann auswählen! (Kriterien siehe Vorgehensweise) -->
+<!-- Warum sind die Klassen existent? Wenn das zu beantworten ist - u good! -->
+<!-- ausgewählte Klassen mit Link, andere einklammern und CRC-Karte löschen -->
+- [Suchen](crc-{Suchen}.md)
+- [Speichern](crc-{Speichern}.md)
+- [Aendern](crc-{Aendern}.md)
+- (Sortieren)
+
+---
+
+# Analyse - Speichermedium
+<!-- Hier Notizen zum Denkprozess! -->
+Das Modul/Der Client "Speichermedium" kümmert sich um die Verwaltung der Daten, welche die drei Funktionen dokumentieren/analysieren. Diese werden dann auf Anfrage vom DataClient wieder ausgegben, im gewünschten Format (einzelne Informationen oder Datensätze).
+     
+## Verantwortlichkeiten
+<!-- Wissen, welches verwaltet und angeboten wird, Aktion die angeboten werden, öffentliche Leistung -->
+<!-- "Walkthrough" -> Szenarien zur Anwendung des Systems -->
+<!-- Nichts, was eine andere Klasse machen könnte -->
+<!-- Die Sachen die die Klasse macht -> keiner anderen Klasse geben -->
+<!-- zentrale Verantwortlichkeiten vs verteilt -->
+Speichern:
+- Die Daten aus den Funktionen, sowie der ConfigFile in einem Speichermedium sichern
+Suchen:
+- Informationen dem DataClient auf Anfrage zur Verfügung stellen
+Löschen:
+- Daten entfernen
+
+## Kollaborationen
+<!-- Benutzeranfragen an Dienste, die benötigt werden um Veranwortlichkeiten zu erfüllen -->
+<!-- enthüllen Kontroll- und Informationsflüsse, und somit Subsysteme -->
+<!-- Können fehlende Verantwortlichkeiten offenbaren, bzw. fehlerhaft zugewiesene -->
+DataClient:
+- Gibt ihm mit der Klasse Suchen, die Informationen weiter
+Drei Funktionen und Config:
```

Comment:
Sorry bin etwas verwirrt, soll jetzt ein direkter Austausch mit Config stattfinden? Beim CRC zu Speichern steht das nicht drin und wenn es irgendwo auftauchen soll, dann vor allem da, glaube ich. @joshuajeschek du meintest irgenwie, dass es maximal in den Notizen auftauchen soll?

Reply:
Ja, ich meinte dass man erstmal Config gar nicht erwähnt, erst ganz am Ende. Also weder in den Kollaborationen, als auch in den Notizen die weiter unten stehen.
Das macht die Notizen und Kollaborationen etwas übersichtlicher, und dann steht da auch nur das drin, was zurzeit klar ist
Die Config-Geschichte würde ich erst ganz am Ende der Notizen anmerken, so wie [hier](https://github.com/weichware-10/meilensteine/tree/crc-config/meilenstein_02/analyse_config#notizen)